benefit/data/sheets/WorkBasketInputFields/WorkBasketInputFields-sandl-nonprod.json

```diff
@@ -1,15 +1,15 @@
 [
-  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-dwpresponsewriter", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting Officer attending?", "DisplayOrder": 10},
-  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting Officer attending?", "DisplayOrder": 18},
-  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-judge", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting Officer attending?", "DisplayOrder": 10},
-  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-registrar", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting Officer attending?", "DisplayOrder": 12},
-  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting Officer attending?", "DisplayOrder": 18},
+  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-dwpresponsewriter", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting officer attending", "DisplayOrder": 11},
+  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting officer attending", "DisplayOrder": 19},
+  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-judge", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting officer attending", "DisplayOrder": 10},
+  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-registrar", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting officer attending", "DisplayOrder": 12},
+  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting officer attending", "DisplayOrder": 18},
```

Comment:
Why the removal of the captial O but not the capital P?
I think I prefer it with a capital O as it is a title.
The '?' should stay.

Reply:
PO approved this change already

Reply:
Remember they don't look at the PRs so don't know to look for changes that aren't on the ticket. I will approve but I recommend you confirm this with a BA first.